index.html

```diff
@@ -0,0 +1,12 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
```

Comment:
- [ ] Please your Pull Request **Summary/description** is not professionally done. Kindly edit it to reflect this project. Eg. 
- Description: In this pull request, I added the following: Using bullet points, list the things you did.